claim-check/code-samples/sample-2/client-consumer/EventHubsConsumer.cs

```diff
@@ -6,66 +6,25 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Microsoft.Azure.EventHubs;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Blob;
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Processor;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Specialized;
 using Newtonsoft.Json.Linq;
 
 namespace ClientConsumer
 {
-    class PartitionReceiver : IPartitionReceiveHandler
-    {
-        readonly CloudBlobClient _cloudBlobClient = null;
-        readonly string _downloadDestination = string.Empty;
-        int _maxBatchSize = 1;
-        public int MaxBatchSize { get => _maxBatchSize; set => _maxBatchSize = value; }
-
-        public PartitionReceiver(CloudStorageAccount storageAccount, string downloadDestination)
-        {
-            _cloudBlobClient = storageAccount.CreateCloudBlobClient();
-            _downloadDestination = downloadDestination;
-        }
-
-        public Task ProcessErrorAsync(Exception error)
-        {
-            Console.WriteLine(error.Message);
-            return Task.FromException(error);
-        }
-
-        public async Task ProcessEventsAsync(IEnumerable<EventData> events)
-        {
-            foreach(var e in events)
-            {
-                string body = Encoding.UTF8.GetString(e.Body);
-
-                var jsonMessage = JArray.Parse(body).First;
-
-                Uri uploadedUri = new Uri(jsonMessage["data"]["url"].ToString());
-                Console.WriteLine("Blob available at: {0}", uploadedUri);
-                var cloudBlob = _cloudBlobClient.GetBlobReferenceFromServer(uploadedUri);
-
-                string uploadedFile = Path.GetFileName(jsonMessage["data"]["url"].ToString());
-                string destinationFile = Path.Combine(_downloadDestination, Path.GetFileName(uploadedFile));
-                Console.WriteLine("Downloading to {0}...", destinationFile);
-                await cloudBlob.DownloadToFileAsync(destinationFile, FileMode.Create);
-                Console.WriteLine("Done.");
-                Console.WriteLine();
-            }
-        }
-    }
-
     class EventHubsConsumer : IConsumer
```

Comment:
I'm a bit concerned with the name of the class, as it is very close to the type `EventHubConsumerClient` from the library.   Looking through the entry point in `Program`, I was momentarily confused.   I think this is exacerbated by this class being build around the `EventProcessorClient` and not the consumer client; their use is very different and this name may cause confusion between the concepts/clients.

Maybe consider something like `EventReader` or `EventProcessorManager` or similar?